modules/planning/tasks/deciders/path_assessment_decider/path_assessment_decider.cc

```diff
@@ -60,12 +60,13 @@ Status PathAssessmentDecider::Process(
   } else {
     ADEBUG << "There are " << candidate_path_data.size() << " candidate paths";
   }
+  auto end_time0 = std::chrono::system_clock::now();
 
   // 1. Remove invalid path.
   std::vector<PathData> valid_path_data;
   for (const auto& curr_path_data : candidate_path_data) {
-    RecordDebugInfo(curr_path_data, curr_path_data.path_label(),
-                    reference_line_info);
+    // RecordDebugInfo(curr_path_data, curr_path_data.path_label(),
```

Comment:
Why comment this out?

Reply:
Only need to turn this on when debugging. Otherwise, too many paths will be displayed.

Reply:
@panjiacheng There is a gflag to turn on/off debug in case you want it. https://github.com/ApolloAuto/apollo/blob/d8182ee0758d8fb5ef7e5acd75aa9538b87e3316/modules/planning/common/planning_gflags.cc#L506

Reply:
Got it. Thx. Since Jinyun needs this pr. I will merge first then update later.